apps/modernization-ui/src/apps/page-builder/components/CreateQuestion/CreateQuestion.tsx

```diff
@@ -345,388 +377,412 @@ export const CreateQuestion = ({ modalRef, question }: any) => {
     const isDisableUnitType = relatedUnits !== 'Yes';
 
     return (
-        <div className="create-question">
-            <div className="create-question__container">
-                <Form onSubmit={onSubmit}>
-                    <div>
-                        <div className="ds-u-text-align--center margin-bottom-2em">
-                            <h3 className="header-title margin-bottom-2px" data-testid="header-title">
-                                {question?.id ? `Edit question` : `Let's create a new question`}
-                            </h3>
-                            <label className="fields-info">
-                                All fields with <span className="mandatory-indicator">*</span> are required
-                            </label>
-                        </div>
-                        <h4>Basic information</h4>
-                        <Controller
-                            control={control}
-                            name="codeSet"
-                            render={({ field: { onChange, value } }) => (
-                                <div className="radio-group">
-                                    <Radio
-                                        id="reportableCondition_Y"
-                                        name="codeSet"
-                                        value={QuestionRequest.codeSet.LOCAL}
-                                        label="LOCAL"
-                                        onChange={(e: any) => onChange(e.target.value)}
-                                        checked={value === 'LOCAL'}
+        <PageBuilder page="question-library">
```

Comment:
Suggestion: The primary reason for the `PageBuilder` wrapper is to add the nav and add a bit of padding to the main `library` components. It is also being phased out due different pages requiring different styling. I would recommend not using it here.